kpet/misc.py

```diff
@@ -32,3 +32,11 @@ def raise_action_not_found(action, command):
             command
         )
     )
+
+
+def raise_invalid_database(database):
+    """Raise an exception for a wrong database"""
+    raise Exception(
+        '"{}" is not a database directory. '
+        'Maybe you need the --db option?'.format(database)
```

Comment:
Could you instead say "Use the --db option to specify the database location"? This way the user will less likely need to read the documentation to figure it out, and won't wonder if they really need the option.

Reply:
Or "the database directory", or something along those lines.